README.md

```diff
@@ -1 +1,26 @@
-# threat-intelligence
\ No newline at end of file
+# threat-intelligence
+
+Infoblox's [Threat Intelligence Group (TIG)](https://www.infoblox.com/cyber-threat-intelligence/) detects, curates, and
+publishes threat intelligence data pertaining to relevant cyber campaigns. While majority of Infoblox's threat intelligence
+is disclosed [privately](https://www.infoblox.com/products/threat-intelligence/) to paying customers, TIG shares 
+indicators of compromise (IOCs) related to threats that are of high interest to the cyber security community through
+this public repository. The following contains descriptions about the contents of each dataset (i.e data folder). All files
+are csv formatted and [MISP](https://www.misp-project.org/) compatible.
+
+
+### ukraine
+This folder contains IOCs related to the Russia-Ukraine conflict. Majority of the content has been organically generated
+using both internal and private data. The `ukraine_russia_legitimate_iocs.csv` file contains confirmed IOCs that are
+managed by legitimate organizations. This includes domains belonging to well-known donation sites providing support to
+Ukrainian civilians, or even newly created support programs operated by entities positively acknowledged by the 
+online community. Oppositely, `ukraine_russia_malicous_suspicious_iocs.csv` contains malicious and suspicious IOCs
+that can cause harm to businesses and innocent users. TIG recommends blocking traffic from network indicators described 
+in this file.
+
+###### Schema Table
+| Field       | Description |
+| ----------- | ----------- |
+| type        | The data type of the IOC. Possible options: domain, ip, hostname, host, url, sha256, email       |
+| indicator   | Also known as an IOC, this analysis artifact is a piece of forensic data related to online activities regarding the Russia-Ukraine conflict.|
+| context     | Descriptive labels that explain the nature of the IOC |
+| source      | The lead we used to find the IOC. If the IOC is derived from a public source, this field will contain a reference link. |
```

Comment:
i think this source is generally not the lead; but a reference we used in checking the lead. i'd make this references. and then include notes.  for context we'll have classification.